app/src/js/customization.js

```diff
@@ -0,0 +1,59 @@
+let head1 = document.getElementById("panel1");
+let head2 = document.getElementById("panel2");
+let head3 = document.getElementById("panel3");
+let head4 = document.getElementById("panel4");
+let body = document.getElementById("main");
+let footer_new = document.getElementById("footer");
+
+let headbgcolor = document.getElementById("headbgcolor");
+let bodybgcolor = document.getElementById("bodybgcolor");
+let footerbgcolor = document.getElementById("footerbgcolor");
+let font = document.getElementById("font");
+
+if (!localStorage.getItem("headbgcolor")) {
+  populateStorage();
+} else {
+  setStyles();
+}
+
+function populateStorage() {
+  localStorage.setItem(
+    "headbgcolor",
+    document.getElementById("headbgcolor").value
+  );
+  localStorage.setItem(
+    "bodybgcolor",
+    document.getElementById("bodybgcolor").value
+  );
+  localStorage.setItem(
+    "footerbgcolor",
+    document.getElementById("footerbgcolor").value
+  );
+  localStorage.setItem("font", document.getElementById("font").value);
```

Comment:
@alok722 , I think it is better to go with https://developer.chrome.com/extensions/storage instead of HTML5 localStorage.